src/rt/sections_win64.d

```diff
@@ -63,9 +63,8 @@ void initSections()
 {
     _sections._moduleGroup = ModuleGroup(getModuleInfos());
 
-    auto pbeg = cast(void*)&__xc_a;
-    auto pend = cast(void*)&_deh_beg;
-    _sections._gcRanges[0] = pbeg[0 .. pend - pbeg];
+    _sections._gcRanges[0] = findImageSection(".data");
```

Comment:
No, there is an unfortunate name collision here. Object file sections ".data" and ".bss" both end up in the image file section ".data".